spec/unit/webrtc/groupCall.spec.ts

```diff
@@ -14,85 +14,305 @@ See the License for the specific language governing permissions and
+            const groupCall = new GroupCall(mockClient, room, GroupCallType.Video, false, GroupCallIntent.Prompt);
+
+            room.currentState.members[FAKE_USER_ID_1] = {
+                userId: FAKE_USER_ID_1,
+            } as unknown as RoomMember;
+
+            await groupCall.create();
+
+            try {
+                await groupCall.enter();
+
+                expect(mockSendState.mock.lastCall[0]).toEqual(FAKE_ROOM_ID);
+                expect(mockSendState.mock.lastCall[1]).toEqual(EventType.GroupCallMemberPrefix);
+                expect(mockSendState.mock.lastCall[2]['m.calls'].length).toEqual(1);
+                expect(mockSendState.mock.lastCall[2]['m.calls'][0]["m.call_id"]).toEqual(groupCall.groupCallId);
+                expect(mockSendState.mock.lastCall[2]['m.calls'][0]['m.devices'].length).toEqual(1);
+                expect(
+                    mockSendState.mock.lastCall[2]['m.calls'][0]['m.devices'][0].device_id,
+                ).toEqual(FAKE_DEVICE_ID_1);
+            } finally {
+                groupCall.leave();
+            }
+        });
+
+        it("starts with mic unmuted in regular calls", async () => {
+            const room = new Room(FAKE_ROOM_ID, mockClient, FAKE_USER_ID_1);
+            const groupCall = new GroupCall(mockClient, room, GroupCallType.Video, false, GroupCallIntent.Prompt);
+
+            try {
+                await groupCall.create();
+
+                await groupCall.initLocalCallFeed();
+
+                expect(groupCall.isMicrophoneMuted()).toEqual(false);
+            } finally {
+                groupCall.leave();
+            }
+        });
+
+        it("starts with mic muted in PTT calls", async () => {
+            const room = new Room(FAKE_ROOM_ID, mockClient, FAKE_USER_ID_1);
+            const groupCall = new GroupCall(mockClient, room, GroupCallType.Video, true, GroupCallIntent.Prompt);
+
+            try {
+                await groupCall.create();
+
+                await groupCall.initLocalCallFeed();
+
+                expect(groupCall.isMicrophoneMuted()).toEqual(true);
+            } finally {
+                groupCall.leave();
+            }
+        });
+
+        it("unmutes audio", async () => {
+            const room = new Room(FAKE_ROOM_ID, mockClient, FAKE_USER_ID_1);
+            const groupCall = new GroupCall(mockClient, room, GroupCallType.Video, false, GroupCallIntent.Prompt);
+
+            try {
+                await groupCall.create();
+
+                await groupCall.initLocalCallFeed();
+
+                await groupCall.setMicrophoneMuted(true);
+
+                expect(groupCall.isMicrophoneMuted()).toEqual(true);
 
-        const mockClient = {
-            sendStateEvent: mockSendState,
-            groupCallEventHandler: {
-                groupCalls: new Map(),
-            },
-        } as unknown as MatrixClient;
+                expect(mockSendState.mock.lastCall[0]).toEqual(FAKE_ROOM_ID);
```

Comment:
It's a bit unclear what this is asserting - is it the call member event? and how does that impact muting/unmuting audio?

Reply:
Good point, I'm not sure what that was there for either: must have been left in by accident.